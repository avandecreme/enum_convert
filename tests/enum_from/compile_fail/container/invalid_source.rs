use enum_convert::EnumFrom;

// enum Source {
//     Unit,
// }

#[derive(EnumFrom)]
#[enum_from(Source)] // There is no Source enum
enum Target {
    Unit,
}

fn main() {}
