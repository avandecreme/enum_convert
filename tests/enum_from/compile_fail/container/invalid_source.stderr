error[E0412]: cannot find type `Source` in this scope
 --> tests/enum_from/compile_fail/container/invalid_source.rs:8:13
  |
8 | #[enum_from(Source)] // There is no Source enum
  |             ^^^^^^ not found in this scope
  |
help: consider importing this struct
  |
1 + use core::error::Source;
  |
